docs/troubleshoot/verify-fingerprint.md

```diff
@@ -121,9 +129,8 @@ USERNAME:/u/username/hash: >
 #### Troubleshooting and hints
 
 This is a worst-case scenario of a bad mismatch.  To find out what the problem is, you could, for example, start by checking the [`manifest.json` file](troubleshoot-zowe-release.md#check-the-zowe-release-number) to see whether one of the components is from the wrong release.
```

Comment:
This displays as
```
start by
checking the Check the Zowe release number on page 347
```
You could instead say
```
start by
referring to Check the Zowe release number on page 347

Reply:
done